CHANGELOG.rst

```diff
@@ -2,7 +2,7 @@
 Changelog
 *********
 
-2.1.0 (TBA)
+2.1.0 (2025-04-08)
```

Comment:
```suggestion
2.0.2 (2025-04-08)
```

Reply:
We are dropping support for Py3.8.

Reply:
It's still backwards-compatible. Pylint didn't bump the feature digit when they dropped 3.8 in 3.3.5 either.